### DISCUSSION TITLE
sysml requirements export and import/update plugin

### DISCUSSION THREAD

**Discussion by mikekidner** [gaphor/gaphor#3613] [Show and tell]
@amolenaar / @danyeaw 
@jashangills and I have been working on a plugin for export/import of requirements which we think is almost ready for putting on pypi. 

It allows users to import/create requirements from an excel workbook, one package per worksheet, and also export to the same. This allows refinement and development of requirements to be done outside Gaphor, which is easier for some project stakeholders, without losing synch with the model. 

The export, includes info about the requirement associations, making the requirements traceability matrix easy to create.

Would we be able to link to it from the gaphor docs? and if so, is there a review process to make sure it's not total rubbish?

**Comment by danyeaw** [comment]
Hi @mikekidner, this sounds like a really nice plugin. We would love to link to plugins from the docs. If you want we can try it out or give it a review when you submit the PR, but highlighting cool things from the community is more important than gatekeeping some type of plugin quality standard, plugins can always improve over time. Cheers!
